chain/client/src/sync.rs

```diff
@@ -853,7 +853,10 @@ impl StateSync {
                 new_shard_sync_download.downloads[0].state_requests_count += 1;
                 new_shard_sync_download.downloads[0].last_target = Some(target.clone());
                 let run_me = new_shard_sync_download.downloads[0].run_me.clone();
-                actix::spawn(
+                near_performance_metrics::stats::spawn(
```

Comment:
shouldn't this be guarded by the feature flag?

Reply:
There is a flag inside near_performance_metrics::stats::spawn